### PR TITLE
Add portia.1.3

### PR DESCRIPTION
**Pull request by rixed**

### CODE REVIEW

File: packages/portia/portia.1.3/opam [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    cat README.md
+    @}
+"""
+build: [
+  [make]
+  [make "check"] {with-test}
+  [make "doc"] {with-doc}
+]
+install: [make "install"]
+remove: ["ocamlfind" "remove" "portia"]
```
Comment (kit-ty-kate) [inline packages/portia/portia.1.3/opam]:
```suggestion
```
Remove field is unnecessary since opam 2.0

File: packages/portia/portia.1.3/opam [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+  "batteries"
+  "qtest" {with-test}
+]
+depexts: [
+  ["asciidoc"] {os-family = "debian"}
+]
+post-messages: [
+  "This package requires asciidoc to build the doc." {failure}
+]
+flags: light-uninstall
```
Comment (kit-ty-kate) [inline packages/portia/portia.1.3/opam]:
```suggestion
```

**Comment by kit-ty-kate** [comment]
Other than those two things, the package seems ok.

**Comment by rixed** [comment]
Thank you @kit-ty-kate for the review.
I just applied the suggested changes.

**Comment by kit-ty-kate** [comment]
Thanks!

**Pull request merged**